cc/tflite_micro/generated/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
I think generated code should also go to the `third_party` directory.
And it doesn't need a `generated` directory name. Could just be `third_party/tflite-micro` (similar to how the corresponding repo is called https://github.com/tensorflow/tflite-micro)

Reply:
+1 since the license is not Oak, it must go under third_party, we don't have a choice.

Reply:
Is it mandatory?
I though about it at the very first place and realized generated sources have a slightly different file hierarchy, put them under third_party seems conflicting with actual source code cloned from tflm repo.

Reply:
I eventually moved it to third_party and created update_tflm.sh to update/downgrade it by needs.